packages/graph/hash_graph/tests/integration/postgres/mod.rs

```diff
@@ -0,0 +1,150 @@
+mod data_type;
+
+use error_stack::Result;
+use graph::{
+    datastore::{
+        DatabaseConnectionInfo, DatabaseType, Datastore, InsertionError, PostgresDatabase,
+        QueryError, UpdateError,
+    },
+    types::{schema::DataType, AccountId, BaseUri, Qualified, VersionId},
+};
+use sqlx::{Connection, PgConnection};
+use tokio::runtime::Runtime;
+use uuid::Uuid;
+
+pub struct Database {
```

Comment:
I don't have a good suggestion but I think we should rename this. I was a little unclear as to what it was in the integration tests and I thought we were maybe testing `graph::datastore::PostgresDatabase` and I was confused by the signatures.

Perhaps we should call this DatabaseTestWrapper or something?

Reply:
Renamed to `DatabaseTestWrapper`: 216820eec0f37b959595b2aaccfb6edb442742e5